File: PHP/1177.php
<?php
// Problema: 1177 - Preenchimento de Vetor II | Resposta: Accepted
// Linguagem: PHP (8.1.2) [+1s]               | Tempo: 0.141s

$v = intval(trim(fgets(STDIN)));
for ($i = 0; $i < 1000; $i++) {
    $num = $i % $v;
    echo "N[{$i}] = {$num}" . PHP_EOL;
}
?>

File: C/1177.c
// Problema: 1177 - Preenchimento de Vetor II | Resposta: Accepted
// Linguagem: C99 [+0s]                       | Tempo: 0.000s

#include <stdio.h>

int main() {
    int v, i;
    scanf("%d", &v);

    for (i = 0; i < 1000; i++) {
        printf("N[%d] = %d\n", i, i % v);
    }

    return 0;
}